src/coreclr/src/vm/pefile.inl

```diff
@@ -1398,8 +1279,7 @@ inline void PEAssembly::GetDisplayName(SString &result, DWORD flags)
     }
 
 #else
-    IMDInternalImport *pImport = GetMDImport();
-    GetFullyQualifiedAssemblyName(pImport, TokenFromRid(1, mdtAssembly), result, flags);
+    result.Set(m_sTextualIdentity);
```

Comment:
Isn't `m_sTextualIdentity` only set under a `_DEBUG` ifdef?

Reply:
Good catch. I have deleted it. We have `m_pDebugName` already that should be good enough as debugging aid.